_config.yml

```diff
@@ -11,7 +14,6 @@ exclude:
 - dap.md
 - uxtesting.md
 - ".sass-cache"
-- nginx.conf
```

Comment:
It's important to not exclude `nginx.conf` from the Jekyll build, as it will need to be inside the `_site` directory for the `staticfile-buildpack` to pick up on it during the deploy process.